CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (cli) [#1362](https://github.com/evmos/ethermint/pull/1362) Fix `index-eth-tx` error when the indexer db is empty.
 * (state) [#1320](https://github.com/evmos/ethermint/pull/1320) Fix codehash check mismatch when the code has been deleted in the evm state.
 * (rpc) [#1392](https://github.com/evmos/ethermint/pull/1392) Allow fill the proposer address in json-rpc through tendermint api, and pass explicitly to grpc query handler.
+* (rpc) [#]() Pass chain-id to grpc query through request, otherwise it's not initialized if abci event don't happens.
```

Comment:
```suggestion
* (rpc) [#1405](https://github.com/evmos/ethermint/pull/1405) Pass chain-id to grpc query through request, otherwise it's not initialized if abci event don't happens.
```